src/feature/gui/gui-runner.c

```diff
@@ -656,6 +656,9 @@ void mGUIRun(struct mGUIRunner* runner, const char* path) {
 				runner->core->reset(runner->core);
 				break;
 			case RUNNER_SAVE_STATE:
+				// If we are saving state, then the screenshot stored for the state previously should no longer be considered up-to-date.
+				// Therefore, mark it as stale, so that at draw time, we will re-draw to the new save state's screenshot.
```

Comment:
This phrasing is a bit funky. Not super important, but I might change it a bit like so:
```suggestion
				// Therefore, mark it as stale so that at draw time we load the new save state's screenshot.
```